### PR TITLE
Update to angular 2.4.6

### PR DESCRIPTION
**Pull request by SimonSchick**
Not much to report, mostly linting due to new tslint and stuff.

### CODE REVIEW

File: src/input.service.ts [hunk 1]
```diff
@@ -396,8 +397,9 @@ export class InputService {
     this.focus.setRoot(root);
 
     this.subscriptions.push(
-      Observable.fromEvent<FocusEvent>(document, 'focusin', { passive: true })
-        .subscribe(ev => this.focus.onFocusChange(<HTMLElement>ev.target))
+      Observable.fromEvent<FocusEvent>(
+        document, 'focusin', { passive: true })
+        .subscribe(ev => this.focus.onFocusChange(<HTMLElement>ev.target)),
```
Comment (UnwrittenFun) [inline src/input.service.ts]: This is now very confusingly formatted, hard to read.

Comment (SimonSchick) [inline src/input.service.ts]: accidental change, sry

**Comment by UnwrittenFun** [comment]
This should be good to merge now

**Comment by SimonSchick** [comment]
cool, thanks for taking over here.

**Pull request merged**